app/controllers/rents_controller.rb

```diff
@@ -9,11 +9,25 @@ def create
     rent = Rent.create(rent_params)
     if rent.valid?
       render json: RentSerializer.new.serialize(rent).to_json, status: :created
+      book = Book.find(rent.book_id)
+      book.times_rented += 1
+      book.save
     else
       render json: { error: rent.errors.messages }, status: :bad_request
     end
   end
 
+  def book_rankings
+    books = Book.all.order(times_rented: :desc)
+    render_paginated books, each_serializer: BookRankingSerializer
+  end
+
+  def active
+    today = DateTime.current.to_date
+    rents = Rent.where("start_date < '#{today}' AND end_date > '#{today}' AND returned = false")
```

Comment:
Move this query logic to model, and you should use and scope to define the query 
https://guides.rubyonrails.org/active_record_querying.html